### PR TITLE
Rename DisableAnimations to UseTransitions

### PR DESCRIPTION
**Pull request by nickrandolph**
GitHub Issue (If applicable): #552

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?

This PR was prematurely merged https://github.com/unoplatform/uno.toolkit.ui/pull/553 with DisableAnimations as the property name

## What is the new behavior?

UseTransitions is more consistent with the useTransitions parameter passed to the GoToState method
Also switched visual states of both LoadingView and ExtendedSplashScreen to use VisualTransition instead of visual state storyboard so that useTransition will skip the animation.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 					xmlns:utu="using:Uno.Toolkit.UI"
 					mc:Ignorable="d">
 
-	<x:String x:Key="DefaultLoadingViewAnimationDuration">00:00:00.083</x:String>
+	<x:String x:Key="DefaultExtendedSplashScreenAnimationDuration">00:00:00.083</x:String>
```
Comment (kazo0) [inline src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml:8]: If you want to make this customizable through something like [Lightweight Styling](https://learn.microsoft.com/en-us/windows/apps/design/style/xaml-styles#lightweight-styling), we can define this in ThemeDictionaries and instead use the `ThemeResource` binding in the VisualStates. So people could override this value without needing to redefine the style.

Sort of like here: https://github.com/unoplatform/uno/blob/9d9619f2af3b5b86b8c0f03a8ee9e5538d2b88f7/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/SplitView_themeresources.xaml#L7-L41

File: src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml
Comment (Xiaoy312) [inline src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml:1]: no IsHitTestVisible="False" on SplashScreenPresenter?

**Review by Xiaoy312 — APPROVED** [review]

**Review by kazo0 — APPROVED** [review]

File: src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml (hunk 1 above)
Comment (nickrandolph) [reply src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml]: done

File: samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/LoadingViewSample.xaml
[the remark this answers is not in the document]
Comment (nickrandolph) [reply samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/LoadingViewSample.xaml:1]: done

File: src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml
Comment (nickrandolph) [reply src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml:1]: This should be set on both loading and splash presenter to ensure they don't conflict with whatever is in the main content

**Pull request merged**